src/utils/memoization.js

```diff
@@ -145,6 +145,10 @@ export function compareLists(listA, listB, comparator = Object.is) {
     return true
```

Comment:
can we a comment for this to say we purposefully want to re-render if the falsey states change